libs/core/src/lib/resizable-card-layout/README.md

```diff
@@ -0,0 +1,24 @@
+# RatingIndicator
+
+This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 11.2.14.
+
+## Code scaffolding
+
+Run `ng generate component component-name --project resizable-card-layout` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project resizable-card-layout`.
+> Note: Don't forget to add `--project resizable-card-layout` or else it will be added to the default project in your `angular.json` file. 
+
+## Build
+
+Run `ng build resizable-card-layout` to build the project. The build artifacts will be stored in the `dist/` directory.
+
+## Publishing
+
+After building your library with `ng build resizable-card-layout`, go to the dist folder `cd dist/resizable-card-layout` and run `npm publish`.
```

Comment:
Dist path here is incorrect. I would suggest removing this content at all since it's a default angular documentation on how to build the lib

Reply:
have removed publishing part for now.